src/components/generic-elements/visualize-button/visualize-button.js

```diff
@@ -8,7 +8,7 @@ import routes from '../../../routes'
 import { GeneratedLink } from '../generated-link'
 
 export default ({ organizationName, datasetName, systemName }) => (
-  <GeneratedLink path={routes.datasetVisualizationView} params={{ organizationName, datasetName }} queryStringParams={{ systemName }}>
-    <div className='visualize-button'>Visualize<InlineSVG style={{ 'marginLeft': '.3rem' }} svg={chart} height='inherit' width={'25'} accessibilityDesc='Chart' /></div>
+  <GeneratedLink path={routes.datasetView} params={{ organizationName, datasetName }} queryStringParams={{ systemName, selectedIndex: 1 }}>
+    <div className='visualize-button'>Visualize<InlineSVG style={{ 'marginLeft': '.3rem' }} svg={chart} height='inherit' width={'25px'} accessibilityDesc='Chart' /></div>
```

Comment:
do we want this size to be in explicit pixels or in rems?

Reply:
We generally use rem to make it easier to swap out sizing based on the root body font, so this is fine

Reply:
Right now we are not using the visualize button because it is being replaced by the tabs but we can remove this so anyone using it in the future can style it how they like